external/fv3fit/fv3fit/_shared/models.py

```diff
@@ -305,12 +324,26 @@ def load(cls, path: str) -> "OutOfSampleModel":
 
         base_model = io.load(config["base_model_path"])
         novelty_detector = io.load(config["novelty_detector_path"])
+        cutoff = 0
```

Comment:
```suggestion
        cutoff = config.get("cutoff", 0)
```
then can omit following if "cutoff" statement